var/spack/repos/builtin/packages/quantum-espresso/configure_aocc.patch

```diff
@@ -2,14 +2,14 @@ diff --git a/install/configure b/install/configure_aocc
 index 66337d1..d2c04af 100755
 --- a/install/configure
 +++ b/install/configure_aocc
-@@ -3203,6 +3203,7 @@ case "$arch" in
+@@ -3199,6 +3199,7 @@
+         ifort_version=`$mpif90 -V 2>&1 | grep "Intel(R)"`
+         pgf_version=`$mpif90 -V 2>&1 | grep "^pgf"`
+         nvfortran_version=`$mpif90 -V 2>&1 | grep "^nvfortran"`
++        aoccflang_version=`$mpif90 -v 2>&1 | grep "AMD clang version"`
+         gfortran_version=`$mpif90 -v 2>&1 | grep "gcc version"`
```

Comment:
Does this patch work for both 6.7 and 6.8? What happens without this patch?

Reply:
The patch is used only when the AOCC compiler is used with QE versions 6.7 and 6.8 (see recipe), so that Spack knows the compiler version.
I have tested the patch by compiling both QE versions, and it works correctly.

The reason for the change is a few changes in the configure script in QE, so that the old diff specification (in terms of reference/surrounding lines) would only work with 6.7 but not 6.8.